src/ui/components/DismissibleTextForm/styles.scss

```diff
@@ -1,30 +1,64 @@
+@import "~photon-colors/colors";
+@import "~amo/css/inc/vars";
+@import "~core/css/inc/mixins";
+@import "~core/css/inc/vars";
 @import "~ui/css/vars";
 
+.DismissibleTextForm-form {
+  margin: 6px 0;
+
+  @include respond-to(medium) {
+    margin: 12px 0;
+  }
+}
+
 .DismissibleTextForm-buttons {
   display: flex;
   justify-content: space-between;
+  margin-top: 6px;
+
+  @include respond-to(medium) {
+    justify-content: center;
+    margin-top: 24px;
+
+    .Button {
+      margin-left: 48px;
+      padding: 0 64px;
```

Comment:
Is this supposed to centre the buttons? In the mocks it looks like they should be centered... though I'd argue that's worse for usability. Hacing them be farther away from each other prevents accidental taps (mainly on touch devices) and having them be on the edges of the screen makes it easier to tap when on a phone/tablet.

I see in the mocks it's:
<img width="376" alt="screenshot 2017-10-08 13 58 34" src="https://user-images.githubusercontent.com/90871/31317006-f0af1b56-ac30-11e7-98b8-7432b40beb52.png">

I also just think it looks a bit off. @pwalm: Could we align theses on the left/right instead of the centre?

Reply:
This adds left and right padding to make the buttons wide like the mocks have them.